src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -137,6 +146,15 @@ export function slickGridDataItemColumnValueWithNoData(value: any, columnDef: an
 	if (displayValue instanceof Array) {
 		displayValue = displayValue.toString();
 	}
+	if (displayValue['ariaLabel']) {
```

Comment:
For these kind of things I find it useful to use type guard check functions like this : https://github.com/microsoft/azuredatastudio/blob/main/src/sql/base/browser/ui/table/formatters.ts#L49

That was the compiler will know the type in the if block and you can access the types directly instead of having to use the indexers.